food-truck/src/app/pages/create-event/create-event.component.css

```diff
@@ -1,3 +1,53 @@
+#map {
+        height: 100%;
+      }
+      /* Optional: Makes the sample page fill the window. */
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 0;
+      }
+    </style>

```

Comment:
Probably shouldn't have tags in here like this, right?

Reply:
corrected